keras_cv/models/backbones/vgg19/vgg19_backbone.py

```diff
@@ -0,0 +1,147 @@
+from keras_cv.models.backbones.vgg16.vgg16_backbone import apply_vgg_block
+from keras_cv.models.backbones.backbone import Backbone
+from keras_cv.models.backbones.vgg19.vgg19_backbone_presets import (
+    backbone_presets,
+)
+from keras_cv.utils.python_utils import classproperty
+
+@keras.utils.register_keras_serializable(package="keras_cv.models")
+class VGG19Backbone(Backbone):
+    """
+    Reference:
+    - [Very Deep Convolutional Networks for Large-Scale Image Recognition](https://arxiv.org/abs/1409.1556) (ICLR 2015)
+    This class represents a Keras VGG19 model.
+    Args:
+      include_rescaling: bool, whether or not to Rescale the inputs.If set to True,
+        inputs will be passed through a `Rescaling(1/255.0)` layer.
+      input_shape: tuple, optional shape tuple, defaults to (224, 224, 3).
+      input_tensor: Tensor, optional Keras tensor (i.e. output of `layers.Input()`)
+        to use as image input for the model.
+    
+    ```python
+    input_data = tf.ones(shape=(3, 224, 224, 3))
+
+    # Randomly initialized backbone with a custom config
+    model = VGG19Backbone(
+        include_rescaling=False,
+    )
+    output = model(input_data)
+    ```
+    """
+
+    def __init__(
+        self,
+        *,
+        include_rescaling,
+        input_tensor=None,
+        input_shape=(224, 224, 3),
+        **kwargs,
+    ):
+
+        inputs = utils.parse_model_inputs(input_shape, input_tensor)
+        x = inputs
+
+        if include_rescaling:
+            x = layers.Rescaling(1 / 255.0)(x)
+
+        x = apply_vgg_block(
+            x=x,
+            num_layers=2,
+            filters=64,
+            kernel_size=(3, 3),
+            activation="relu",
+            padding="same",
+            max_pool=True,
+            name="block1",
+        )
+
+        x = apply_vgg_block(
+            x=x,
+            num_layers=2,
+            filters=128,
+            kernel_size=(3, 3),
+            activation="relu",
+            padding="same",
+            max_pool=True,
+            name="block2",
+        )
+
+        x = apply_vgg_block(
+            x=x,
+            num_layers=4,
+            filters=256,
+            kernel_size=(3, 3),
+            activation="relu",
+            padding="same",
+            max_pool=True,
+            name="block3",
+        )
+
+        x = apply_vgg_block(
+            x=x,
+            num_layers=4,
+            filters=512,
+            kernel_size=(3, 3),
+            activation="relu",
+            padding="same",
+            max_pool=True,
+            name="block4",
+        )
+
+        x = apply_vgg_block(
+            x=x,
+            num_layers=4,
+            filters=512,
+            kernel_size=(3, 3),
+            activation="relu",
+            padding="same",
+            max_pool=True,
+            name="block5",
+        )
+
+        super().__init__(inputs=inputs, outputs=x, **kwargs)
+
+        self.include_rescaling = include_rescaling
```

Comment:
In order to use this as a Backbone for Object detection, we'll also need to setup self.pyramid_level_outputs.

The pyramid_level_outputs should be a dict with keys 1...N where the values are the last intermediate outputs of the model where the size of the Tensor has been reduced by 2^N.

(So, for example, if the input is 512x512, the key 1 would point to the output of the last intermediate res block whose output is 256x256, and the key 3 would point to the output of the last intermediate res block whose output is 64x64)

I would guess that this will be somewhat straightforward for VGG, because we can create `pyramid_level_outputs = {}` before all of the `apply_vgg_block`s, and then after each block we can just say `pyramid_level_outputs[block_index] = x`. But I'm not certain.